scripts/SetupServiceFabric/SetupServiceFabric.sh

```diff
@@ -21,37 +21,31 @@ ExitIfError()
 }
 
 Distribution=`lsb_release -cs`
-if [ "xenial" != "$Distribution" ]; then
+if [[ "xenial" != "$Distribution" && "bionic" != "$Distribution" ]]; then
```

Comment:
Did this work on both Ubuntu-16 and 18 ?

Reply:
yup, checked on both

Reply:
Thanks